editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -254,21 +247,19 @@ pub fn _transform_properties(document_node: &DocumentNode, node_id: NodeId, _con
 		{
 			widgets.extend_from_slice(&[
 				WidgetHolder::unrelated_separator(),
-				WidgetHolder::new(Widget::NumberInput(NumberInput {
-					value: Some(vec2.x),
-					label: "X".into(),
-					unit: " px".into(),
-					on_update: update_value(move |number_input: &NumberInput| TaggedValue::DVec2(DVec2::new(number_input.value.unwrap(), vec2.y)), node_id, index),
-					..NumberInput::default()
-				})),
+				NumberInput::new()
+					.value(Some(vec2.x))
```

Comment:
Perhaps also `value` should be part of the `new()` constructor for NumberInputs? (Or all *Inputs).